index.html

```diff
@@ -1,15 +1,182 @@
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="#">English</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="#">My page</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="#">logout</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <nav class="navbar nav-menu navbar-expand-lg">
+    <button type="button" class="navbar-toggler button-nav border-0" data-toggle="collapse" data-bs-target="#navbarContent" aria-controls="navbarContent" aria-expanded="false">
+      <span class="navbar-toggler-icon" id="button-toggle"></span>
+    </button>
+  
+    <a class="navbar-brand img-brand" href="#"><img src="img/ruby-1.png" class="img-logo" alt=""></a>
+    <div class="collapse navbar-collapse pb-3 show" id="navbarContent">
+      <ul class="navbar-nav nav-list pe-0">
+        <li class="nav-item">
+          <a class="nav-link ms-4 fs-6 text-first" aria-current="page" href="about.html">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link ms-4 fs-6 text-nav" href="#">Program</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link ms-4 fs-6 text-nav" href="#">Join</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link ms-4 fs-6 text-nav" href="#">Sponsor</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link ms-4 fs-6 text-nav" href="#">News</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link ms-4 text-first text-nav nav-border border border-3 border-danger" href="#">CC Campaign</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <header class="content pb-5">
+    <div class="content-1">
+      <h1 class="header-content">Web Design and Web Development Global Summit<br>2022 Kenya</h1>
+      <p class="textone border border-white border-2">A joyful celebration believing in the value of openness and shairing, creating<br>
+        a positive change with people from over 80 countries is taking place in October, in Kenya.
+      </p>
+    </div>
+    <div class="headers">
+      <h1 class="text-center fw-bold h-font">2022.10.20(THUR) ~ 22(SAT)</h1>
+      <h2 class="text-center fs-4 fw-light">@USIU - Ampitheatre hall</h2>
+    </div>
+  </header>
+  <main class="program">
+    <h1 class="text-white text-center">Main Program</h1>
+    <div class="bar"></div>
+  <div class="container">
+    <div class="list">
+      <div class="list-icon">
+        <img src="img/program_icon_01.png" alt="program-icon">
+      </div>
+      <div class="subtitle">
+        <p class="subtitle-p fs-4">conference</p>
+      </div>
+      <div class="description text-white">
+        Listen to speakers from various countries and about the latest trends in the world.
+      </div>
+    </div>
+    <div class="list">
+      <div class="list-icon">
+        <img src="img/program_icon_02.png" alt="program-icon">
```

Comment:
- [ ] Kindly here there's a button you have missed kindly add it to match the original layout.

![image](https://user-images.githubusercontent.com/35535130/181568738-94ce5f0b-e15b-4095-8a72-307ee4655496.png)

Reply:
Okay, but you have also to make it appears on the desktop version, to match the original design.